### PR TITLE
Update alert messages and conditional statement

### PR DESCRIPTION
**Pull request by nbarnabee**
While it's unlikely that any of these messages will ever be seen, they now better indicate what would have happened to cause them.

This addresses [T333086]( https://phabricator.wikimedia.org/T333086)

### CODE REVIEW

File: src/components/UserContributionForm.vue [hunk 1]
```diff
@@ -51,19 +51,22 @@ const submit = async () => {
...
-          if (error.response.status === 401) {
+          } else if (error.response.status === 401) {
             alert(
               "You must be logged in to submit a task. Please click on the Login button."
             );
-          }
-          alert(
-            "There was a problem sending your request. Please try again later."
-          );
+          } else if (error.response.status === 400) {
```
Comment (Damilare1) [inline src/components/UserContributionForm.vue:62]: should we use a swich statement or better still an object for this response status options?

**Review by Damilare1 — COMMENTED** [review]
The text changes look good to me, just a small suggestion inline to reduce space and improve code readability. Please let me know your thoughts.

File: src/components/UserContributionForm.vue (hunk 1 above)
Comment (nbarnabee) [reply src/components/UserContributionForm.vue]: Addressed with [1ee079a](https://github.com/wikimedia/toolhunt-ui/pull/57/commits/1ee079aae2f807905904859fb1b65ad8d1ed020a)

Wow, that is a lot cleaner!

**Review by Damilare1 — APPROVED** [review]
LGTM! Thanks

**Pull request merged**